src/MongoDB.Analyzer/DiagnosticsRules.cs

```diff
@@ -25,6 +25,11 @@ internal static class DiagnosticsRules
     private const string DiagnosticIdBuilders2MQL = "MABuilders1001";
     private const string DiagnosticIdNotSupportedBuilderExpression = "MABuilders2001";
 
+    //EF
+    private const string DiagnosticIdEF2MQL = "MAEF1001";
```

Comment:
Needs also to update `AnalyzerReleases.Shipped.md`

Reply:
Done